R/fit-models.R

```diff
@@ -99,48 +94,17 @@ build_formula <- function(target, covariates, smooth_terms = NULL, group_by = "g
 #'                    k = "auto")
 #' }
 fit_gam <- function(df_tract,
-                    target = NULL,
```

Comment:
I think that we do want to allow target to be NULL for the case that the user provides a formula.

This is the test that is currently failing on the CI.